src/main/java/transforms/CombineDateTimeTransform.java

```diff
@@ -79,22 +77,24 @@ public Object apply( Map<String, String> row ) {
             return null;
         }
         final JavaDateTimeHelper tHelper = new JavaDateTimeHelper( this.timezone,
-                timePattern );
+                this.timePattern, this.shouldAddTimezone );
 
         LocalTime time;
         try {
             time = tHelper.parseTime( t );
         } catch ( Exception e ) {
-            time = dHelper.parseDateTimeAsTime( d );
+            time = tHelper.parseDateTimeAsTime( t );
         }
 
-        if ((date == null) || (time == null)){return null;};
+        if ( ( date == null ) || ( time == null ) ) {return null;}
+        ;
 
         // combine
-        if ( date  == null |  time == null ) {
+        if ( date == null | time == null ) {
```

Comment:
I think this is supposed to be `||`, but this whole check is redundant from above and should be removed